target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -35,6 +35,13 @@ engenius,eap1300)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:yellow:wlan5g" "phy1tpt"
 	ucidef_set_led_default "mesh" "MESH" "${boardname}:blue:mesh" "0"
 	;;
+engenius,eap2200)
+	ucidef_set_led_wlan "wlan2g" "WLAN2G" "${boardname}:blue:wlan2g" "phy0tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:yellow:wlan5g" "phy1tpt"
+	ucidef_set_led_wlan "wlan5g2" "WLAN5G2" "${boardname}:yellow:wlan5g2" "phy2tpt"
+	ucidef_set_interface_lan "eth0"
+	ucidef_set_interface_lan "eth1"
```

Comment:
`ucidef_set_interface_lan` belongs to `02_network`, not here.

Reply:
And it's already done there, so those two lines can just be removed here.

Reply:
Thanks for advice.